python/test/t_CovarianceModel_std.expout

```diff
@@ -201,3 +201,12 @@ parameterDescription= [scale_0,scale_1,amplitude_0,amplitude_1,amplitude_2]
 myModel( [-1,-2] ,  [3,5] )= class=SquareMatrix dimension=3 implementation=class=MatrixImplementation name=Unnamed rows=3 columns=3 values=[0.0170869,0,0,0,4.67048e-05,0,0,0,0.000156845]
 dCov = class=Matrix implementation=class=MatrixImplementation name=Unnamed rows=2 columns=9 values=[0.00683477,0.0113913,0,0,0,0,0,0,2.98911e-05,0.000145304,0,0,0,0,0,0,6.77907e-05,0.000329538]
 dCov (FD)= class=Matrix implementation=class=MatrixImplementation name=Unnamed rows=2 columns=9 values=[0.00683478,0.0113913,0,0,0,0,0,0,2.98911e-05,0.000145306,0,0,0,0,0,0,6.77908e-05,0.000329542]
+myModel =  class=IsotropicCovarianceModel input dimension=2, dimension = 1, kernel =class=SquaredExponential scale=class=Point name=Unnamed dimension=1 values=[3.5] amplitude=class=Point name=Unnamed dimension=1 values=[1.5]
+active= [0,1]
+parameter= [3.5,1.5]
+parameterDescription= [scale_0,amplitude_0]
+myModel( [-1,-2] ,  [3,5] )= class=SquareMatrix dimension=1 implementation=class=MatrixImplementation name=Unnamed rows=1 columns=1 values=[0.158479]
+dCov = class=Matrix implementation=class=MatrixImplementation name=Unnamed rows=2 columns=1 values=[-0.0517484,-0.0905596]
+dCov (FD)= class=Point name=Unnamed dimension=2 values=[0.0517484,0.0905598]
```

Comment:
@josephmure there is a bug here! The sign of the gradient is wrong!!!
I fix it locally and push it soon with many other fixes

Reply:
Oh yes, sorry! To fix it, you will need to change
https://github.com/openturns/openturns/blob/1493931eb60cc7497e02675d09152b17a2197833/lib/src/Base/Stat/IsotropicCovarianceModel.cxx#L112

and write 
```
 const Matrix kernelPartialGradient(Point(1), kernel_.partialGradient(Point(1, tauNorm))); 
```
instead.

Reply:
no. We should consider `tau = (s-t)` and not `tau = (t-s)`. See conventions
Already fixed (with other bugs). Coming soon ;-)

Reply:
Also, this `test_model` function could be improved: it should use `ott.assert_almost_equal` to compare the `partialGradient` result with the finite differences result.

Reply:
yep this is how I tracked it!
I am relying on `assert_almost_equal ` instead of useless `print`